r4/smartconfig/tests/test_service.bal

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/test;
+
+@test:Config {groups: ["Service"]}
+function testService() returns error? {
+    http:Client testClient = check new ("http://localhost:9090");
+    json|error response = testClient -> /fhir/r4/\.well\-known/smart\-configuration;
```

Comment:
```suggestion
    json|error response = testClient -> /fhir/r4/[WELL_KNOWN]/smart\-configuration;
```

Define a constant and use in other places as well. We can use this approach to write a readable code when a resource path segment lengthy and complex.
```bal
const WELL_KNOWN = ".well-known";

Reply:
We can do the same in the resource method declaration as well

Reply:
Unfortunatly service seems to be not working with this apporach. I met with the following error continously.

`no matching resource found for path : /fhir/r4/wellknown/smart-configuration , method : GET" error = "no matching resource found for path : /fhir/r4/wellknown/smart-configuration , method : GET" stackTrace = []`

Reply:
Seems its a but in ballerina. https://github.com/ballerina-platform/ballerina-standard-library/issues/4556

Reply:
Ack